policyengine_us/variables/gov/states/ar/tax/income/ar_taxable_capital_gains.py

```diff
@@ -10,12 +10,22 @@ class ar_taxable_capital_gains(Variable):
     defined_for = StateCode.AR
 
     def formula(person, period, parameters):
-        long_term_capital_gains = person("long_term_capital_gains", period)
-        short_term_capital_gains = person("short_term_capital_gains", period)
+        net_lt_capital_gains = person("long_term_capital_gains", period)
+        net_st_capital_gains = person("short_term_capital_gains", period)
+        # Can be capital loss
+        net_cap_gain = net_lt_capital_gains - net_st_capital_gains
```

Comment:
what line does this correspond to? i don't see where short-term net gains would be subtracted from long-term net gains

Reply:
Line 7a - the naming net lt gains was wrong - fixed